google/cloud/bigtable/internal/rpc_policy_parameters.inc

```diff
@@ -0,0 +1,89 @@
+// Define the defaults using a pre-processor macro, this allows
+// the application developers to change the defaults for their
+// application by compiling with different values.
+
+#ifndef BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY
+#define BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY std::chrono::milliseconds(5)
+#endif  // BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY
+
+#ifndef BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+#define BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD \
+  std::chrono::milliseconds(600000)
+#endif  // BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+
+#ifndef BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY
+#define BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY \
+  std::chrono::milliseconds(60000)
+#endif  // BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY
+
+RPCPolicyParameters const kBigtableInstanceAdminLimits = {
+    BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY,
+    BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD,
+    BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY,
+};
+
+// Interface name: "google.bigtable.admin.v2.BigtableTableAdmin"
+// Define the defaults using a pre-processor macro, this allows
+// the application developers to change the defaults for their
+// application by compiling with different values.
+
+#ifndef BIGTABLETABLEADMIN_DEFAULT_INITIAL_DELAY
+#define BIGTABLETABLEADMIN_DEFAULT_INITIAL_DELAY std::chrono::milliseconds(100)
+#endif  // BIGTABLETABLEADMIN_DEFAULT_INITIAL_DELAY
+
+#ifndef BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+#define BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD \
+  std::chrono::milliseconds(600000)
+#endif  // BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+
+#ifndef BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_DELAY
+#define BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_DELAY \
+  std::chrono::milliseconds(60000)
+#endif  // BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_DELAY
+
+RPCPolicyParameters const kBigtableTableAdminLimits = {
+    BIGTABLETABLEADMIN_DEFAULT_INITIAL_DELAY,
+    BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD,
+    BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_DELAY,
+};
+
+// Interface name: "google.bigtable.v2.Bigtable"
+// Define the defaults using a pre-processor macro, this allows
+// the application developers to change the defaults for their
+// application by compiling with different values.
+
+#ifndef BIGTABLE_DEFAULT_INITIAL_DELAY
+#define BIGTABLE_DEFAULT_INITIAL_DELAY std::chrono::milliseconds(100)
+#endif  // BIGTABLE_DEFAULT_INITIAL_DELAY
+
+#ifndef BIGTABLE_DEFAULT_MAXIMUM_RETRY_PERIOD
+#define BIGTABLE_DEFAULT_MAXIMUM_RETRY_PERIOD std::chrono::milliseconds(600000)
+#endif  // BIGTABLE_DEFAULT_MAXIMUM_RETRY_PERIOD
+
+#ifndef BIGTABLE_DEFAULT_MAXIMUM_DELAY
+#define BIGTABLE_DEFAULT_MAXIMUM_DELAY std::chrono::milliseconds(60000)
+#endif  // BIGTABLE_DEFAULT_MAXIMUM_DELAY
+
+RPCPolicyParameters const kBigtableLimits = {
+    BIGTABLE_DEFAULT_INITIAL_DELAY,
+    BIGTABLE_DEFAULT_MAXIMUM_RETRY_PERIOD,
+    BIGTABLE_DEFAULT_MAXIMUM_DELAY,
+};
```

Comment:
Consider adding a`#undef` for each one of the macros at the end of this file.  Once used to define the C++ constants they are no longer needed and we should avoid polluting the application namespace with macros.

Reply:
`#undef` added at the end of file.